tools/model_explorer_circle/src/model_explorer_circle/main.py

```diff
@@ -172,13 +172,13 @@ def build_graph(self, me_graph: graph_builder.Graph) -> None:
         # Create pseudo const node for orphan tensors (= const tensors)
         for tensor_id, tensor in enumerate(sub_graph.tensors):
             if (self.get_source_of(tensor_id)) is None:
-                me_node = graph_builder.GraphNode(id=f'{input_id + tensor_id}',
+                me_node = graph_builder.GraphNode(id=f'{input_id + tensor_id + 1}',
```

Comment:
optional) can we use something like
```suggestion
                node_id = input_id + tensor_id + 1
                me_node = graph_builder.GraphNode(id=f'{node_id}',
```
?

Reply:
Yes, I like the way you suggested ;)
The reason why I didn't add temporary variable (`node_id`) is because of pylint's warning.
It warns that there're already too many local variables in `build_graph()` (15 is recommended, but it's 17 now 🙄)

Reply:
Perhaps in the future, we can extract more common codes  and that will result in a reduction in the number of local variables.
I'll apply the changes for now.

Reply:
>  is because of pylint's warning.

wow...